test/test_sync.py

```diff
@@ -554,3 +558,78 @@ def test_sync_without_references(mc, project_name: str, config_update: dict):
     sql = "SELECT count(*) FROM notes WHERE ext_url IS NULL"
     gpkg_cur.execute(sql)
     assert gpkg_cur.fetchone()[0] == 3
+
+
+def test_google_drive_backend(mc):
+    """Test media sync connected to google drive backend"""
+    project_name = "mediasync_test_googledrive"
+    full_project_name = WORKSPACE + "/" + project_name
+    work_project_dir = os.path.join(TMP_DIR, project_name + "_work")
+
+    cleanup(mc, full_project_name, [work_project_dir])
+    prepare_mergin_project(mc, full_project_name)
+
+    # patch config to fit testing purposes
+    config.update(
+        {
+            "MERGIN__USERNAME": API_USER,
+            "MERGIN__PASSWORD": USER_PWD,
+            "MERGIN__URL": SERVER_URL,
+            "MERGIN__PROJECT_NAME": full_project_name,
+            "PROJECT_WORKING_DIR": work_project_dir,
+            "OPERATION_MODE": "copy",
+            "REFERENCES": [
+                {
+                    "file": None,
+                    "table": None,
+                    "local_path_column": None,
+                    "driver_path_column": None,
+                }
+            ],
+            "DRIVER": "google_drive",
+            "GOOGLE_DRIVE__SERVICE_ACCOUNT_FILE": GOOGLE_DRIVE_SERVICE_ACCOUNT_FILE,
+            "GOOGLE_DRIVE__FOLDER": GOOGLE_DRIVE_FOLDER,
+            "GOOGLE_DRIVE__SHARE_WITH": "jan.caha@lutraconsulting.co.uk",
+        }
+    )
+
+    google_drive_delete_folder(GoogleDriveDriver(config), GOOGLE_DRIVE_FOLDER)
+
+    main()
```

Comment:
There is quite a few extra methods in driver, I wonder if we should not test also some basic failures, even with mocked api or something.

Reply:
ok, if we have positive and negative test, we can leave it out